include/z64item.h

```diff
@@ -343,71 +378,153 @@ typedef enum {
     /* 0x8F */ GI_MASK_KAFEIS_MASK,
     /* 0x91 */ GI_CHATEAU = 0x91,
     /* 0x92 */ GI_MILK,
-    /* 0x93 */ GI_GOLD_DUST,
-    /* 0x94 */ GI_EEL, // Unused bottled eel
-    /* 0x95 */ GI_SEAHORSE, 
+    /* 0x93 */ GI_GOLD_DUST_2,
+    /* 0x94 */ GI_HYLIAN_LOACH, // Unused bottled eel
+    /* 0x95 */ GI_SEA_HORSE_CAUGHT,
     /* 0x96 */ GI_MOON_TEAR,
     /* 0x97 */ GI_DEED_LAND,
     /* 0x98 */ GI_DEED_SWAMP,
     /* 0x99 */ GI_DEED_MOUNTAIN,
     /* 0x9A */ GI_DEED_OCEAN,
-    /* 0x9B */ GI_SWORD_GREAT_FAIRY,
-    /* 0x9C */ GI_SWORD_KOKIRI,
-    /* 0x9D */ GI_SWORD_RAZOR,
-    /* 0x9E */ GI_SWORD_GILDED,
+    /* 0x9B */ GI_SWORD_GREAT_FAIRY_STOLEN,
+    /* 0x9C */ GI_SWORD_KOKIRI_STOLEN,
+    /* 0x9D */ GI_SWORD_RAZOR_STOLEN,
+    /* 0x9E */ GI_SWORD_GILDED_STOLEN,
     /* 0x9F */ GI_SHIELD_HERO_2, // Code that treats this as hero's shield is unused, so take with a grain of salt
```

Comment:
```suggestion
    /* 0x9F */ GI_SHIELD_HERO_STOLEN, // Code that treats this as hero's shield is unused, so take with a grain of salt
```
Hmm, wonder if we should also classify this as stolen. Based on `EnGirla` (where the name comes from in the first place) this would have been sold by the curiosity shop as well, so it makes sense for it to have been stolen.